### DISCUSSION TITLE
用户注册求助

### DISCUSSION THREAD

**Discussion by swordpp** [requarks/wiki#6371] [Help / Questions]
当我在wikijs系统里开放了用户注册后，每当用户输入信息，点击注册以后，就会出现如下提示：
![image](https://user-images.githubusercontent.com/131244293/233010783-fd44a48c-73c7-4846-9932-90b0fa140df8.png)
能不能不出现这个提示，当点击注册按钮以后，直接返回到登陆页面，并且系统自动通过验证？？

**Comment by swordpp** [comment]
还有一个，用户登陆以后如何直接转到用户的首页，而不是创建首页或者管理员页哪个页面？
![image](https://user-images.githubusercontent.com/131244293/233019699-d61d8cd5-2fa3-451a-b4d2-490109c43b5a.png)

**Comment by cdn0x12** [comment]
1. As the system prompt said, your email configuration is incomplete. You need to go to the administration area to configure email-sending settings to make email verification work for registration.

2. Set the "redirect at login" field to `/p` for the corresponding user group.
